src/Orchard.Web/Modules/Orchard.MediaLibrary/Scripts/media-library.js

```diff
@@ -71,8 +71,11 @@ $(function () {
 
         var listWidth = $('#media-library-main-list').width();
         var listHeight = $('#media-library-main-list').height();
-        var itemWidth = $('.thumbnail').first().width();
-        var itemHeight = $('.thumbnail').first().height();
+        var itemWidth = $('#media-library-main-list li').first().width();
+        var itemHeight = $('#media-library-main-list li').first().height();
+        var defaultDimension = $(window).width() < 1420 ? 120 : 200;

```

Comment:
There's a media query that changes the default thumbnail size to 120 on smaller width screens.  This accounts for that.